contrib/swank-asdf.lisp

```diff
@@ -386,13 +386,14 @@ Example:
   "Returns a list of all systems in ASDF's central registry
 AND in its source-registry. (legacy name)"
   (unique-string-list
-   (mapcar
-    #'pathname-name
-    (while-collecting (c)
+   (while-collecting (c)
```

Comment:
This form moved left one column, so its body needs to be reindented. Other than that this looks great.

Reply:
indented now